src/main/webapp/WEB-INF/views/UserManagement.jsp

```diff
@@ -0,0 +1,142 @@
+                                            required />
+                                        <div class="has-error" ng-show="myForm.$dirty">
+                                            <span ng-show="myForm.email.$error.required">This is a required field</span>
+                                            <span ng-show="myForm.email.$invalid">This field is invalid </span>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="row">
+                                <div class="form-actions floatRight">
+                                    <input type="submit" value="{{!userController.user.id ? 'Add' : 'Update'}}"
+                                        class="btn btn-primary btn-sm" ng-disabled="myForm.$invalid">
+                                    <button type="button" ng-click="userController.reset()"
+                                        class="btn btn-warning btn-sm" ng-disabled="myForm.$pristine">Reset
+                                        Form</button>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="panel panel-default">
+                    <!-- Default panel contents -->
+                    <div class="panel-heading"><span class="lead">List of Users </span></div>
+                    <div class="tablecontainer">
+                        <table class="table table-hover">
+                            <thead>
+                                <tr>
+                                    <th>ID</th>
+                                    <th>Name</th>
+                                    <th>Address</th>
+                                    <th>Email</th>
+                                    <th width="20%"></th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr ng-repeat="u in userController.users">
+                                    <td><span ng-bind="u.id"></span></td>
+                                    <td><span ng-bind="u.username"></span></td>
+                                    <td><span ng-bind="u.address"></span></td>
+                                    <td><span ng-bind="u.email"></span></td>
+                                    <td>
+                                        <button type="button" ng-click="userController.edit(u.id)"
+                                            class="btn btn-success custom-width">Edit</button> <button type="button"
+                                            ng-click="userController.remove(u.id)"
+                                            class="btn btn-danger custom-width">Remove</button>
+                                    </td>
+                                </tr>
```

Comment:
interactive elements need unique ids,  use the index available in ng-repeat, to generate unique ids for each row and button.